test/Sentry.DiagnosticSource.IntegrationTests/SqlListenerTests.RecordsEf.Core3_1.verified.txt

```diff
@@ -1,4 +1,4 @@
-[
+﻿[
```

Comment:
I've seen this a couple of times now, what's the diff here?

Reply:
There used to be a [UTF-8 BOM](https://en.wikipedia.org/wiki/Byte_order_mark) at the beginning of the file, and it was deleted during merge.  Why, I'm not sure - but it doesn't affect anything.

Also, GitHub used to show a BOM with a red dot, and for some reason it doesn't anymore.  Can't find any setting to show them or any announcement that they changed something... odd.

Reply:
why was the bom deleted? Verify explicitly write a bom